kolibri/core/assets/src/views/app-body.vue

```diff
@@ -10,8 +10,10 @@
       :delay="false"
     />
     <template v-else>
```

Comment:
You could have replaced the template with the div.

Reply:
good catch - PR here #3817

Reply:
(I see now that's what you wrote above originally)